### PR TITLE
[WIP] Bigbang component

### PR DESCRIPTION
**Pull request by runyontr**
Proposed change to repo in support of Big Bang Noun.

Blocked by this being released: 

https://github.com/defenseunicorns/zarf/pull/1094

Solves:

#22 
#24 
#7 
#6
#12

### CODE REVIEW

File: cocowow/zarf.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        - ../values/kyverno.yaml
+        - ../values/istio.yaml
+        - ../values/loki.yaml
+        - ../values/promtail.yaml
+        - ../values/kiali.yaml
+        - ../values/monitoring.yaml
+        - ../values/neuvector.yaml
+        - ../values/tempo.yaml
+        - ../values/authservice.yaml
+        - ../values/loki.yaml
```
Comment (MxNxPx) [inline cocowow/zarf.yaml:61]: duplicate ../values/loki.yaml

File: defense-unicorns-distro/zarf.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        - ../values/kyverno.yaml
+        - ../values/istio.yaml
+        - ../values/loki.yaml
+        - ../values/promtail.yaml
+        - ../values/kiali.yaml
+        - ../values/monitoring.yaml
+        - ../values/neuvector.yaml
+        - ../values/tempo.yaml
+        - ../values/authservice.yaml
+        - ../values/loki.yaml
```
Comment (MxNxPx) [inline defense-unicorns-distro/zarf.yaml:63]: duplicate ../values/loki.yaml

**Comment by runyontr** [comment]
I was planning on waiting until zarf has a release with the Big Bang noun.  If we update the docs so people know to use the RC, then we can merge and iterate IMO

File: values/minio.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+          mode: PERMISSIVE
+
+  minio:
+    enabled: true
+    values:
+      istio:
+        mtls:
+          mode: PERMISSIVE
+      # https://github.com/minio/operator/blob/master/docs/crd.adoc
+      tenants:
```
Comment (blancharda) [inline values/minio.yaml:26]: I believe this should be `tenant` to match [the BB chart](https://repo1.dso.mil/big-bang/product/packages/minio/-/blob/main/chart/values.yaml#L81)

You may also need to add `podManagementPolicy: "OrderedReady"` in order for the stateful set to come up healthy.

Comment (runyontr) [reply values/minio.yaml:26]: I think there's a good conversation aroudn whether MinIO is needed as part of BB.  Perhaps for non cloud Velero backups?

Comment (blancharda) [reply values/minio.yaml:26]: Loki can also be configured to use it. 

I think it's also going to be relatively common for 3rd party apps to need a bucket for testing. Might be worth keeping around just for that.

Comment (runyontr) [reply values/minio.yaml:26]: In my head, this BB core is what's needed for the platform admins, not for tenants.  If tenants need something we should support adding it, but probably not as part of this distro

Comment (blancharda) [reply values/minio.yaml:26]: Makes sense.

I'm not sure the implications of using the default PVC with Loki vs setting it up with a minio bucket.
Doing so might make it easier to transition to actual S3 buckets?

Maybe @UncleGedd, @rjferguson21, @RothAndrew have more informed thoughts?

**Pull request merged**